lib/eventFields.js

```diff
@@ -6,7 +6,7 @@ import {
   GraphQLSchema,
 } from 'graphql';
 
-import { EventFetch, AllEventsFetch } from './eventFetch.js';
+import { fetchOne, fetchAll } from './eventOrNewsFetch.js';
```

Comment:
@jameshopkins do you suggest we do a document type split in there? There will be more document types soon.